### DISCUSSION TITLE
Feedback on Documentation - Request for User-Friendly Examples

### DISCUSSION THREAD

**Discussion by BMUZAMMIL** [community/community#60180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

I hope this email finds you well. I wanted to share some feedback regarding the documentation on your website. As an average web designer, I thought it would be valuable to provide my perspective based on my personal experience with the site.

While the majority of the documentation is commendable, I believe it may not cater to the needs of an average person. Many individuals, like myself, often seek practical "code" examples that are easy to copy, paste, and modify according to their specific requirements. Unfortunately, the current examples provided on the site do not fulfill this need.

In comparison, I have found the Bootstrap website documentation, along with other similar sites, to be highly helpful and conducive to learning, especially for beginners. Therefore, I would like to suggest a solution to address this issue. Creating [codepen.io](http://codepen.io/) examples and embedding them within the documentation, or alternatively, providing direct links to relevant examples, would greatly enhance the user experience. This approach would allow most people to grasp the concepts more easily, as not everyone possesses advanced knowledge of JavaScript and HTML.

To further illustrate my point, I spent several hours figuring out the necessary JavaScript and how it all fits together. Eventually, I managed to create a functional example that I believe demonstrates the functionality I was aiming for. I have shared it on CodePen, and you can access it using the following link: [ https://codepen.io/DesignersWeb/pen/OJamJOb ].

Lastly, I would like to express my gratitude for providing such an outstanding slider. Its quality and performance are truly impressive.

Thank you for considering my feedback. I believe implementing user-friendly examples will greatly benefit both novice and experienced users alike. I look forward to seeing further improvements on your website.

Best regards,
Bmuzammil

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
